AppCenterDemoApp/Assets/Puppet/PuppetAppCenter.cs

```diff
@@ -223,8 +223,6 @@ public void SetCustomDialog(bool enable)
 
     public void SetStartupMode(int startupMode)
     {
-        // A workaround for android. Both calls to PlayerPrefs and AndroidUtility are needed.
-        // AndroidUtility allows to load preferenes in native Java code, PlayerPrefs is for Unity layer.
```

Comment:
Why it was deleted?
cc @annakocheshkova is it correct?

Reply:
Probably not...